src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -277,20 +277,6 @@ public static void TestDepthBeyondLimit(int depth)
             { }
         }
 
-        [Theory]
-        [InlineData(0)]
-        [InlineData(-1)]
-        public static void TestDepthInvalid(int depth)
```

Comment:
Is there a replacement for this test that I'm not seeing?

Reply:
Yes. In JsonReaderStateTests.